xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1047 @@
+         </para>
+         <itemizedlist>
+          <listitem>
+           <para>
+            SLES 15
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            2 (v)CPU
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            8 GB RAM
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            500 GB Storage
+           </para>
+          </listitem>
+         </itemizedlist>
+        </listitem>
+       </itemizedlist>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+    <important>
+     <title>Adjust Number Of Mirror Servers</title>
+     <para>
+      This scenario description does not contain any fallback contingencies for
+      the mirror servers. Add additional mirror servers (behind a loadbalancer)
+      if you require additional reliability/availability.
+     </para>
+    </important>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.network">
+    <title>Networking</title>
+    <para>
+     All members of the &productname; cluster must be able to communicate with
+     the internal mirror server(s) within the airgapped network.
+    </para>
+    <para>
+     Ports:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       80 HTTP - RMT Server and Helm chart repository mirror
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       443 HTTPS - RMT Server and Helm chart repository mirror
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       5000 HTTP - Docker registry
+      </para>
+     </listitem>
+    </itemizedlist>
+    <note>
+     <title>Additional Port Configuration</title>
+     <para>
+      If you choose to add more Docker registries to your internal network,
+      these must run on different ports than the standard registry running on
+      <literal>5000</literal>.
+     </para>
+    </note>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.storage">
+    <title>Trusted Storage</title>
+    <para>
+     Transferring data from the external network mirror to the internal mirror
+     can be performed in many ways. The most common way is portable storage (USB
+     keys or external hard drives).
+    </para>
+    <para>
+     Sizing of the storage is dependent on the number of data sources that need
+     to be stored. Container images can easily measure several Gigabytes per
+     item; although they are generally smaller for Kubernetes related
+     applications. The overall size of any given RPM repository is at least tens
+     of Gigabytes. For example: At the time of writing, the package repository
+     for &sls; contains approximately <literal>36GB</literal> of data.
+    </para>
+    <para>
+     The storage must be formatted to a filesystem type supporting files larger
+     than 4GB. We recommend <literal>btrfs</literal>.
```

Comment:
Not sure if using `btrfs` for the mirror host is sensible. I would suggest `xfs` but overall I think we could possibly avoid recommending a particular FS since it also depends on disk configuration etc

Reply:
Is xfs natively supported by SLE/opensuse? I don't want to have to add instructions how to get a specific FS working to these instructions. btrfs works out of the box and has >4GB support.

Reply:
XFS is supported by my point mainly is that the FS chose is relevant to other factors such as RAID configuration, HW specs etc which means that `btrfs` may not be the best choice in all cases. That's why I suggested to either drop the suggestion or add a note that admins should also asses the type of FS they need themselves.

Reply:
Ok I'll change the wording. I understand this as a "recommendation" is what we suggest and if you need something else you need to figure that out. It's not "your disk must and can only be formatted to btrfs".